src/datasources/blockchain/blockchain-api.manager.spec.ts

```diff
@@ -32,6 +32,45 @@ describe('BlockchainApiManager', () => {
 
       expect(api).toBe(cachedApi);
     });
+
+    it('should include the INFURA_API_KEY in the RPC URI for an Infura URI with API_KEY_PATH authentication', async () => {
+      const rpcUriValue = `${faker.internet.url()}infura${faker.string.sample()}`;
```

Comment:
Nit: I'd suggest we try to mirror the Infura RPC as closely as possible:

```suggestion
      const rpcUriValue = `https://${faker.string.sample()}.infura.io/v3/`;
      
```